app/src/main/java/io/github/muntashirakon/setedit/Native.java

```diff
@@ -2,8 +2,6 @@
 
 import android.os.Build;
 
-import androidx.annotation.Keep;
```

Comment:
You must keep this annotation, otherwise the app would crash in the production build due d8 optimising this file.

Reply:
I forgot that when I was preparing the merge.